ReactSkia/sdk/RNSKeyCodeMapping.h

```diff
@@ -148,18 +134,33 @@ typedef enum rnsKey{
     RNS_KEY_BraceRight,
     RNS_KEY_BracketLeft,
     RNS_KEY_BracketRight,
-    RNS_KEY_Escape,
-    RNS_KEY_Space,
-    RNS_KEY_Home,
-    RNS_KEY_EndKey,
-    RNS_KEY_Insert,
-    /*Arthematic */
+        /*Arthematic */
     RNS_KEY_Minus,
     RNS_KEY_Equal,
     RNS_KEY_Percent,
     RNS_KEY_Plus,
     RNS_KEY_Grater,
     RNS_KEY_Less,
```

Comment:
Minor comment : Can RNS_KEY_Less be changed to  RNS_KEY_Lesser to match with  RNS_KEY_Greater

Reply:
RNS_KEY_Less is kept because the key name should be inline with libwpe and X11 key name. 
    Example X11:
    case XK_greater: return RNS_KEY_Greater;
    case XK_less: return RNS_KEY_Less;
   Example libWPE:
       case WPE_KEY_greater: return RNS_KEY_Greater;
    case WPE_KEY_less : return RNS_KEY_Less;